src/model/document.js

```diff
@@ -161,14 +166,14 @@ export default class Document {
 		// Buffer marker changes.
 		// This is not covered in buffering operations because markers may change outside of them (when they
 		// are modified using `model.markers` collection, not through `MarkerOperation`).
-		this.listenTo( model.markers, 'update', ( evt, marker, oldRange, newRange ) => {
+		this.listenTo( model.markers, 'update', ( evt, marker, oldRange, newRange, affectsData ) => {
```

Comment:
Do we need `affectsData` being passed in the event? Can't we simply use `marker.affectsData`?